UsingIsWhere/Readme.md

```diff
@@ -0,0 +1,35 @@
+# Using isWhere in a VDR
+This is an example of using the where clause in a VDR when the search term is different from the original object (in a GET request, this would mean the vendor field in a VDR). The example VDR uses Netsuite ERP's ledgerAccount object. Although mapping `account_type` to Netsuite's `acctType.value` yields the correct results when executing a GET request without a where clause, it does not work with a where clause because Netsuite is expecting the search term `type` to search for the field `acctType.value`. Using custom javascript in a VDR can transform the where clause as well, so that you can query `account_type='_bank'` and receive the correct results.
```

Comment:
@kiawnna is it worth clarifying that `isWhere` could be used to modify the contents of a where clause?  It's not limited to cases where the search term needs to change.

Reply:
agreed. done. @nick-bair